other/restrict_node_selection/restrict_node_selection.yaml

```diff
@@ -34,8 +34,14 @@ spec:
       - resources:
           kinds:
           - Pod
+    preconditions:
+      all:
+      - key: "{{request.operation || 'BACKGROUND'}}"
+        operator: Equal
```

Comment:
Operator name is `Equals`.

Reply:
Thanks! Hint for me:
Use official docs https://kyverno.io/docs/writing-policies/preconditions and not an existing ClusterPolicy.